src/main/java/com/example/rickmorty/util/impl/JsonParserImpl.java

```diff
@@ -0,0 +1,69 @@
+package com.example.rickmorty.util.impl;
+
+import com.example.rickmorty.model.Episode;
+import com.example.rickmorty.model.Location;
+import com.example.rickmorty.model.RickMortyCharacter;
+import com.example.rickmorty.util.JsonParser;
+import java.util.ArrayList;
+import java.util.List;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.springframework.stereotype.Component;
+
+@Component
+public class JsonParserImpl implements JsonParser {
+    @Override
+    public Integer getQuantityOfCharacters(String json) {
+        JSONObject obj = new JSONObject(json);
+        return obj.getJSONObject("info").getInt("count");
+    }
+
+    @Override
+    public Integer getNumberOfPages(String json) {
+        JSONObject obj = new JSONObject(json);
+        return obj.getJSONObject("info").getInt("pages");
+    }
+
+    @Override
+    public List<RickMortyCharacter> getJsonArray(String json) {
+        JSONObject obj = new JSONObject(json);
```

Comment:
try to parse json to entities with ObjectMapper

Reply:
ObjectMapper is class of jackson library. I use JSON In Java library which uses JSONObject instead.